kolibri/plugins/facility/assets/src/views/DataPage/index.vue

```diff
@@ -249,7 +249,8 @@
         return this.$route.name === PageNames.DATA_EXPORT_PAGE;
       },
       windowSizeStyle() {
-        if (this.windowIsMedium || this.windowIsSmall) {
+        const { windowIsMedium, windowIsSmall } = useKResponsiveWindow();
```

Comment:
It is best to consume the composable via the composition API, so instead of making the modification here, you would add a `setup()` method to the component, like this:

```
setup() {
    const { windowIsMedium, windowIsSmall } = useKResponsiveWindow();
    return { windowIsMedium, windowIsSmall };
},
```

`this.windowIsMedium` and `this.windowIsSmall` will now be defined on the component, as they are returned as properties of the setup function return value.

Reply:
Hi @rtibbles thanks for the hint. I tried to fix it as you suggested.